client/my-sites/media/main.jsx

```diff
@@ -238,7 +238,7 @@ export default React.createClass( {
 					<MediaLibrarySelectedData siteId={ site.ID }>
 						<MediaLibrary
 							{ ...this.props }
-							className="media__main-section"
+							className="media__primary"
```

Comment:
So, it seems this `main.jsx` component should be using `<Main class="media">`, and instead of adding a class like `media__primary` you could do `.media .media-library` which more clearly means "the media library block within the media section".